locales/en.yml

```diff
@@ -68,4 +68,9 @@ en:
         service_not_running: |-
           # %{name} service is not running in the vagrant box.
         nil: |-
-          # Showing the status of providers in the vagrant box:
+          Configured services:
+        status:
+          running:
```

Comment:
This could be `running: "status"` as `|- (or | )` is used for multi-line string.